examples/cluster/plot_cluster_comparison.py

```diff
@@ -27,7 +27,7 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-from sklearn import cluster, datasets
+from sklearn import cluster, datasets, mixture
 from sklearn.neighbors import kneighbors_graph
 from sklearn.preprocessing import StandardScaler
 
```

Comment:
I've said that because I add a warning with my sublime text extension but maybe I've configured it to be hard :).
Flake8 and pep8 don't say anything indeed, sorry for that.